src/components/AnnotationBrowser/AnnotationList.vue

```diff
@@ -180,9 +213,23 @@ export default class AnnotationList extends Vue {
   readonly filterStore = filterStore;
   readonly getStringFromPropertiesAndPath = getStringFromPropertiesAndPath;
 
+  columnOptions = [
+    { text: "Annotation ID", value: "annid" },
+    { text: "Index", value: "id" },
```

Comment:
`annid` is not easy to differentiate from `id`
The issue here is that `id` have been badly named, it should be `index` or at least `idx`
Suggested modification:
- `id` -> `index`
- `annid` -> `id`

Reply:
Agreed, I was scared to make the edit :). I'll do that now.

Reply:
@bruyeret I just pushed a new commit to the branch fixing this problem.